webview_panels/src/modules/app/requestExecutor.ts

```diff
@@ -8,7 +8,7 @@ const requestMap: Record<
 
 export const executeRequestInSync = (
   url: string,
-  params: Record<string, unknown>,
+  params: Record<string, unknown> | Record<string, unknown>[],
```

Comment:
if this is for updating defer config, pls keep as Record<string, unknown> and pass the values as {config: [{key: value}]}